frontend/src/components/messstelle/optionsmenue/panels/MessquerschnittPanel.vue

```diff
@@ -118,10 +119,14 @@ const richtungValues: ComputedRef<Array<KeyVal>> = computed(() => {
     }
     messstelle.value.messquerschnitte.forEach(
         (querschnitt: MessquerschnittInfoDTO) => {
+            let himmelsrichtungAsText = himmelsRichtungenTextLong.get(
+                querschnitt.fahrtrichtung
+            );
+            if (himmelsrichtungAsText === undefined) {
+                himmelsrichtungAsText = querschnitt.fahrtrichtung.valueOf();
```

Comment:
Wenn du in der Map keinen passenden wert findest, dann würde ich hier erwarten das ein ein "Fehler" drinnen steht oder ähnliches. Warum nimmst du hier das `querschnitt.fahrtrichtung.valueOf()`

Reply:
Stimmt. Theoretisch kann es eh nicht zu undefined kommen, da wir die Daten ja vorher in der EAI dann validieren müssen wenn wir sie laden. Ich ändere das auf einen Fehlertext ab